### PR TITLE
Update the Index page of our website and modify sample cards and decks to look like our UG picture

### LINKED ISSUE #296: [PE-D][Tester E] No sample data on initial load
In the documentation, it is stated there will be some sample data. However, when I loaded up the app initially, there isn't any. 

This could simply be a wording issue, where you instead meant that the image of the app contains sample data. That being said, it might be good to rephrase to avoid ambiguity. 

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: zhacatomn/ped#9

### PR DESCRIPTION
**Pull request by rockman007372**
Closes #296

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -1,19 +1,25 @@
 ---
 layout: page
-title: AddressBook Level-3
+title: PowerCards
 ---
 
 [![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
```
Comment (ruiyigan) [inline docs/index.md:6]: Can change this to 
`[CI Status](https://github.com/AY2223S2-CS2103T-W11-3/tp/actions)`

File: docs/index.md [hunk 2]
```diff
@@ -1,19 +1,25 @@
...
 [![codecov](https://codecov.io/gh/se-edu/addressbook-level3/branch/master/graph/badge.svg)](https://codecov.io/gh/se-edu/addressbook-level3)
```
Comment (ruiyigan) [inline docs/index.md:7]: can remove this since we not using codecov and no need for it yet.

**Review by ruiyigan — APPROVED** [review]
LGTM! Just some minor comments.

File: docs/index.md (hunk 1 above)
Comment (rockman007372) [reply docs/index.md:6]: yepp sure

File: docs/index.md (hunk 2 above)
Comment (rockman007372) [reply docs/index.md:7]: Oh we for real gonna discard it forever? I didnt know it was optional

**Pull request merged**